tidb-cloud/data-service-manage-endpoint.md

```diff
@@ -89,9 +89,36 @@ On the right pane of the endpoint details page, you can click the **Properties**
 
 - **Path**: the path that users use to access the endpoint.
 
-    - The combination of the request method and the path must be unique within a Data App.
-    - Only letters, numbers, underscores (`_`), and slashes (`/`) are allowed in a path. A path must start with a slash (`/`) and end with a letter, number, or underscore (`_`). For example, `/my_endpoint/get_id`.
     - The length of the path must be less than 64 characters.
+    - The combination of the request method and the path must be unique within a Data App.
+    - Only letters, numbers, underscores (`_`), slashes (`/`), and parameters enclosed in curly braces (such as `{var}`) are allowed in a path. Each path must start with a slash (`/`) and end with a letter, number, or underscore (`_`). For example, `/my_endpoint/get_id`.
+    - For parameters enclosed in `{ }`, only letters, numbers, and underscores (`_`) are allowed. Each dynamic parameter must start with a letter or (`_`). 
+
+    > **Note:**
+    >
+    > - In a path, each parameter must be at a separate level and does not support prefixes or suffixes.
+    >
+    >    *Valid path*: ```/var/{var}``` and  ```/{var}```
+    >
+    >    *Invalid path*:```/var{var}``` and ```/{var}var```
+    >
+    > - Paths with the same method and prefix might conflict, as in the following example:
+    >
+    >    ```GET /var/{var1}```
+    >
+    >    ```GET /var/{var2}```
+    >
+    >   These two paths will conflict with each other because `GET /var/123` matches both.
+    >     
+    > - Paths with parameters have lower priority than paths without parameters. For example:
+    >
+    >    ```GET /var/{var1}```
+    >
+    >    ```GET /var/123```
+    >
+    >   These two paths will not conflict because `GET /var/123` takes precedence.
+    > - Path parameters can be used directly in SQL. For more information, see [Configure parameters](#configure-parameters).
```

Comment:
```suggestion
    >   These two paths will not conflict because `GET /var/123` takes precedence.
    >
    > - Path parameters can be used directly in SQL. For more information, see [Configure parameters](#configure-parameters).
```